src/components/Key/Key.spec.js

```diff
@@ -23,4 +23,8 @@ describe('Key', () => {
     it('Renders a Key with the correct Key Value', () => {
         expect(subject.text()).toEqual(props.keyValue)
     })
+
+    it('Renders a Key with the correct className', () => {
```

Comment:
```suggestion
    it('renders a key with the correct className', () => {
```
Pedantic, I know, but also how we usually write them here 😛